src/projectM-jack/Makefile.am

```diff
@@ -22,14 +22,13 @@ ${QT_CXXFLAGS} \
 
 # TODO: build qprojectM-jack and projectM-jack as separate programs
 
-bin_PROGRAMS = projectM_jack
+bin_PROGRAMS = projectM-jack
```

Comment:
this is okay i just hope it doesn't trip up downstream maintainers

Reply:
I forgot to mention this also is align with desktop file configuration , which try run projectM-jack